src/controllers/userController.ts

```diff
@@ -0,0 +1,21 @@
+import { findUserByEmail, findUserById, findAllUsers, deleteUserById } from "@/services/userService";
```

Comment:
the name of the imports clash with the names of the functions in the file causing error. Use 'as' keyword to differentiate or change the name of the functions in the file.